app/controllers/images_controller.rb

```diff
@@ -0,0 +1,72 @@
+class ImagesController < ApplicationController
```

Comment:
Great that you included a separate controller for images. This is something that I needed to do myself as I had just implemented it in with posts